frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -294,13 +312,13 @@ export const updateVmSettingsState = (options: UpdateOptions) =>
   [
     selectUserTemplateOnLoadedUpdater,
     selectedUserTemplateUpdater,
-    flavorUpdater,
     osUpdater,
     baseImageUpdater,
     cloneCommonBaseDiskImageUpdater,
     workloadConsistencyUpdater,
     provisioningSourceUpdater,
     nativeK8sUpdater,
+    flavorUpdater,
```

Comment:
please move the code as well. We want to ensure that the updaters can be read from top to bottom in the same order they are executed in